migrator/README.md

```diff
@@ -0,0 +1,261 @@
+# Migrator Tool
+
+The library takes in E.164 phone numbers with their corresponding BCP-47 country code. If inputted numbers match any of the supported internal renumbering ‘recipes’, which are created from official country documentation, said numbers will undergo a process of migration to convert them into the most up to date, dialable format. A given “recipe” is an internal record which holds data needed to migrate a stale number into one that is valid and dialable. 
+
+## Capabilities
+
+```Single Number Migration``` - input a single E.164 phone number with its corresponding BCP-47 country code. If there is an available migration recipe, the number will be converted to the new format based on the given recipe. The resulting string of the migration will be outputted to the console.
```

Comment:
Maybe a blurb at the start of the capabilities section saying that the library can be used as a command-line tool (and maybe link usage docs below?) or a web application (and link it to a web application section). Then we can leave the capabilities as everything here except listing the web application.